client-spark/spark3/src/main/java/org/apache/spark/shuffle/writer/RssShuffleWriter.java

```diff
@@ -563,24 +582,23 @@ private void clearFailedBlockState(ShuffleBlockInfo block) {
     partitionLengths[block.getPartitionId()] -= block.getLength();
   }
 
-  private ShuffleServerInfo reassignFaultyShuffleServer(
-      Set<Integer> partitionIds, String faultyServerId) {
+  private void triggerReassignShuffleServer(
+      Set<Integer> partitionIds, String faultyServerId, Set<Integer> needLoadBalancePartitions) {
```

Comment:
I think `needLoadBalancePartitions` can be simplified to `loadBalancePartitions`, there is  no benefit in adding `need`.

Reply:
Got it.